api/clients/v2/disperser_client.go

```diff
@@ -163,18 +181,34 @@ func (c *DisperserClient) DisperseBlob(
 	// if this is nil, that indicates we will use the legacy payment system to create the paymentMetadata
 	paymentMetadata *core.PaymentMetadata,
 ) (*corev2.BlobHeader, *disperser_rpc.DisperseBlobReply, error) {
+	ctx, span := tracer.Start(ctx, "DisperserClient.DisperseBlob",
```

Comment:
Probably also makes sense to encapsulate tracing logic for the `DisperserClient`